php-classes/Emergence/People/Invitation.php

```diff
@@ -40,6 +43,21 @@ class Invitation extends \ActiveRecord
         ]
     ];
 
+    public static $validators = [
+        'UserClass' => [
+            'validator' => 'selection',
+            'choices' => [],
+            'required' => false
+        ]
+    ];
+
+    public static function __classLoaded()
+    {
+        $fieldCls = User::getStackedConfig('fields', 'Class');
+        static::$validators['UserClass']['choices'] = $fieldCls['values'];
+        parent::__classLoaded();
```

Comment:
@nbey classLoaded functions shouldn't call `parent` as they get called already when the parent class loads. The only case for it is when the current class overrides some things AND overrides other things that the parent __classLoaded sets based on the former things because you want the later overridden things repopulated by the parent logic